Modules/@babylonjs/react-native/windows/BabylonReactNative/EngineViewManager.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+#include "NativeModules.h"
+#include "winrt/Microsoft.ReactNative.h"
+#include "JSI/jsi.h"
+#include <unordered_set>
+
+namespace winrt::BabylonReactNative::implementation {
+
+    struct EngineViewManager
+        : winrt::implements<
+        EngineViewManager,
+        winrt::Microsoft::ReactNative::IViewManager,
+        winrt::Microsoft::ReactNative::IViewManagerWithReactContext,
+        winrt::Microsoft::ReactNative::IViewManagerWithNativeProperties,
+        winrt::Microsoft::ReactNative::IViewManagerWithExportedEventTypeConstants,
+        winrt::Microsoft::ReactNative::IViewManagerWithCommands>
+        , std::enable_shared_from_this<EngineViewManager> {
+    public:
+        EngineViewManager();
+
+        // IViewManager
+        winrt::hstring Name() noexcept;

```

Comment:
nit: for functions like `Name()` that are just getters, make them `const` wherever possible.